examples/struct-person.flix

```diff
@@ -0,0 +1,29 @@
+/// A mutable struct that represents a person
+struct Person[r] {
+    name: String,
+    age: Int32,
+    height: Int32
+}
+
+def giveBirth(name: String, rc: Region[r]): Person[r] \ r =
+    new Person { name = name, age = 0, height = 30 } @ rc
+
+def displayPerson(person: Person[r]): Unit \ {r, IO} =
```

Comment:
Lets drop this and simply inline the print into main.

Reply:
You can also use string interpolation "foo${person.blarg}"to make it smaller.

Reply:
Sorry, did you mean to drop `giveBirth` or `displayPerson`?